### PR TITLE
Issue #678: Add lando tooling to pull and pipe in the latest backdrop…

### PR DESCRIPTION
**Pull request by serundeputy**
…cms.org sanitized database.

### CODE REVIEW

File: config/scripts/lp-pull.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * If you have the creds make sure to set them in your .env file and
+ * do a `lando rebuild -y` to let lando load the env vars into the app.
+ */
+
+$user = getenv('LP_USER');
+$pass = getenv('LP_PASS');
+$db_url = getenv('LP_DB_URL');
+$files_url = getenv('LP_FILES_URL');
+
+if (!isset($user) || !isset($pass)) {
```
Comment (BWPanda) [inline config/scripts/lp-pull.php:30]: I wanted to see this message displayed, so I purposefully didn't create an `.env` file and ran `lando pulldb`, but it didn't show this... Instead I got:
```bash
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
File /app/backdropcmsorg-latest-sanitized.sql.gz not found!
```

File: .gitignore [hunk 2]
```diff
@@ -10,4 +10,13 @@ private-files
 /*.sql.gz
 /*.tgz
 /*.tar.gz
+*.sql
+*.sql.gz
+*.tar
+*.tar.gz
```
[the remark this answers is not in the document]
Comment (serundeputy) [reply .gitignore:16]: I don't know who or why those were put in that way ... so i left them.

File: .lando.yml [hunk 3]
```diff
@@ -19,3 +21,10 @@ tooling:
     cmd: drush --root=/app/www --uri=https://borg.lndo.site
   redis-cli:
     service: cache
+  pulldb:
+    service: appserver
+    description: 'Pull just the latest DB from backdropcms.org'
+    cmd:
+      - appserver: php /app/config/scripts/lp-pull.php --database
```
[the remark this answers is not in the document]
Comment (serundeputy) [reply .lando.yml:28]: that is where I usually put scripts in my projects even before anything called backdrop ever existed.

File: config/scripts/lp-pull.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @file
+ * Get the latest sanitized database for local development.
+ *
+ * This will only work if you have the creds set up in
+ * your .env file.
```
[the remark this answers is not in the document]
Comment (serundeputy) [reply config/scripts/lp-pull.php:7]: If we had linting i'd agree but we don't so this part of the conversation just wastes everyone's time :wink:

File: config/scripts/lp-pull.php (hunk 1 above)
Comment (serundeputy) [reply config/scripts/lp-pull.php:30]: let me see if including the URL vars in this helps.

File: config/scripts/lp-pull.php [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+ * Allotting access to creds will be made on a case by case basis.
+ *
+ * If you have the creds make sure to set them in your .env file and
+ * do a `lando rebuild -y` to let lando load the env vars into the app.
+ */
+
+$user = getenv('LP_USER');
+$pass = getenv('LP_PASS');
+$db_url = getenv('LP_DB_URL');
+$files_url = getenv('LP_FILES_URL');
```
[the remark this answers is not in the document]
Comment (serundeputy) [reply config/scripts/lp-pull.php:26]: yes; LP = lando pull.

**Comment by jenlampton** [comment]
This PR was closed accidentally when master branch was deleted.

**Comment by jenlampton** [comment]
It looks like the last comment above was actually resolved (about adding LP into the docs). Is this okay to merge?

**Pull request merged**